src/api/app/controllers/webui/image_templates/interconnects_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Webui
+  module ImageTemplates
+    class InterconnectsController < Webui::WebuiController
+      def index
+        @projects = Project.image_templates
+        respond_to do |format|
+          format.xml
```

Comment:
@evanrolfe IIRC, our thinking was that it would be nice to get a bit closer to REST instead of splitting the logic for API and WebUI. I still think that's a good idea:-)